src/backend/utils/storage.js

```diff
@@ -69,13 +72,27 @@ module.exports = {
    */
   removeFeed: async (id) => {
     const key = createFeedKey(id);
-    await redis
-      .multi()
-      .hdel(key, 'id', 'author', 'url', 'user', 'link', 'etag', 'lastModified')
-      .srem(feedsKey, id)
-      .exec();
+    // If feed is in t:feeds
+    if (await redis.sismember(feedsKey, id)) {
+      await redis
+        .multi()
+        .hdel(key, 'id', 'author', 'url', 'user', 'link', 'etag', 'lastModified')
+        .srem(feedsKey, id)
+        .exec();
+    } else {
```

Comment:
Follow-up question: if a feed isn't in either set, do we care?  That is, will we do the right thing, or crash?  Do we need another check for membership?

Probably this is fine, but maybe add a test?

Reply:
Will add a test. I don't think it will error, should just return `0` or `false`, but lets see

Reply:
I don't know if it is what you expected, but I added line 277 in `feed.test.js` and line 81 in `storage.test.js`